api/db-pouch.js

```diff
@@ -5,6 +5,7 @@ PouchDB.plugin(require('pouchdb-mapreduce'));
 const { COUCH_URL } = process.env;
 // strip trailing slash from to prevent bugs in path matching
 const couchUrl = COUCH_URL && COUCH_URL.replace(/\/$/, '');
-const DB = new PouchDB(couchUrl);
 
-module.exports.medic = DB;
+module.exports.medic = new PouchDB(couchUrl);
+module.exports.audit = new PouchDB(`${couchUrl}-audit`);
+module.exports._url = couchUrl;
```

Comment:
I hit the same problem but I've solved it differently in my replace kanso branch: https://github.com/medic/medic-webapp/blob/3019-replace-kanso/api/db-pouch.js

I don't know which solution is better but mine doesn't make you mess with the `require` cache.

Reply:
This particular piece of code has nothing to do with messing with the require cache?